lib/matplotlib/tests/test_table.py

```diff
@@ -229,3 +233,30 @@ def test_table_bbox(fig_test, fig_ref):
                   loc='center',
                   bbox=Bbox.from_extents(0.1, 0.2, 0.9, 0.8)
                   )
+
+
+@check_figures_equal()
+def test_table_unit(fig_test, fig_ref):
+    # test that table doesn't participate in unit machinery,\
+    # instead uses repr/str
+
+    class FakeUnit:
+        def __init__(self, thing):
+            pass
+        def __repr__(self):
+            return "Hello"
+
+    fake_convertor = munits.ConversionInterface()
+    fake_convertor.convert = MagicMock(side_effect=lambda v, u, a: 0)
+    fake_convertor.default_units = MagicMock(side_effect=lambda v, a: None)
+    fake_convertor.axisinfo = MagicMock(side_effect=lambda u, a: munits.AxisInfo())
```

Comment:
Do we need a `MagicMock`; can we use a regular `Mock`? I'm sometimes wary of the magic.